package.json

```diff
@@ -55,6 +55,8 @@
     "@types/benchmark": "^1.0.30",
     "@types/jest": "^26.0.0",
     "@types/rdf-js": "^4.0.0",
+    "@types/spark-md5": "^3.0.2",
```

Comment:
In Comunica-related projects we have the convention to add `@types/*` for main dependencies as actual dependency (not dev), could you move these two to below?

Reply:
ah, my bad, I thought these had ended there by mistake

Reply:
FYI, this is why: https://github.com/comunica/comunica/issues/294